### ISSUE TITLE
fix: Optional payload in action create necessary parameter

### ISSUE THREAD

**Issue by smoonsf** [danielroe/nuxt-typed-vuex#48]
**🐛 The bug**
```js
export const actions = actionTree(
  { state, mutations, getters },
  {
    doSomething({commit}, payload?: string) { // optional parameter
      // ...do something
    }
  }
);
```
`$accessor.login();` create typescript error

**🛠️ To reproduce**

**🌈 Expected behaviour**
`$accessor.login();` not to create ts error.

**ℹ️ Additional context**

**Comment by danielroe** [comment]
@smoonsf Appreciate the bug report! Should be fixed in `0.1.13`, but do let me know if you experience any other issues :smiley:

**Comment by smoonsf** [comment]
@danielroe  Appreciate for good repo. :) Thanks for quick fix.
